plugins/modules/aci_syslog_group.py

```diff
@@ -246,17 +293,20 @@ def main():
     argument_spec.update(aci_annotation_spec())
     argument_spec.update(
         name=dict(type="str", aliases=["syslog_group", "syslog_group_name"]),
-        format=dict(type="str", choices=["aci", "nxos"]),
+        format=dict(type="str", choices=["aci", "nxos", "rfc5424-ts", "enhanced_log"]),
+        local_file_log_format=dict(type="str", choices=["aci", "nxos", "rfc5424-ts", "enhanced_log"]),
+        console_log_format=dict(type="str", choices=["aci", "nxos", "rfc5424-ts", "enhanced_log"]),
```

Comment:
should we define constant for the `["aci", "nxos", "rfc5424-ts", "enhanced_log"]` one time, perhaps in constant file? if defined as dict `LOG_FORMATS={"enhanced_log": "rfc5424-ts", "nxos": "nxos", "aci":"aci", "rfc5424-ts":"rfc5424-ts"}` you can do use the list equivalent of this dict. Similar like you have done in other modules.

furthermore this allows you to simplify the argument retrieval part where you code below

```
console_format = module.params.get("console_log_format")
console_log_format = "rfc5424-ts" if console_format == "enhanced_log" else console_format
```

could be reduced to 

```
console_log_format = LOG_FORMATS.get(module.params.get("console_log_format"))
```